tests/metrics/test_metastatus_lib.py

```diff
@@ -1037,3 +1037,19 @@ def test_get_mesos_disk_status():
     }
     actual = metastatus_lib.get_mesos_disk_status(metrics)
     assert actual == (100, 50, 50)
+
+
+def test_reserved_maintenence_resources_no_maintenenance():
+    actual = metastatus_lib.reserved_maintenence_resources({})
+    assert all([actual[x] == 0 for x in ['cpus', 'mem', 'disk']])
+
+
+def test_reserved_maintenence_resources():
```

Comment:
Can you also add a simple test that has resources reserved for the `maintenance` role plus some other bogus role?